src/beast/test/asio/streambuf.cpp

```diff
@@ -0,0 +1,296 @@
+//
```

Comment:
You can cherry pick this commit to get boost build working on linux. Running `b2` built without warnings: https://github.com/seelabs/rippled/commit/de2494011d6cf3794fbc31b1ba1d4c9ff86dfd6d

I still have not resolved the `/usr/share/boost-build` conflict.